src/inverseRobot.js

```diff
@@ -23,8 +23,18 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
-}
+  const object = {};
 
+  for (const key in robot) {
+    object[robot[key]] = key;
+  }
+
+  if (Object.keys(object).length < Object.values(robot).length) {
+    return null;
+  } else {
+    return object;
+  }
```

Comment:
If you would check copies in you loop(and return null there), then you could always return object here.